chapters/en/Unit 1 - Fundamentals/imaging.mdx

```diff
@@ -0,0 +1,29 @@
+1. Image Acquisition Fundamentals in Digital Processing
+Image acquisition in digital processing is a foundational step in transforming physical phenomena into digital representations. It starts with the interaction between an illumination source and the subject being imaged. This illumination can be of various types, from conventional light sources to more sophisticated forms like electromagnetic or ultrasound energy. The interaction results in energy either being reflected from or transmitted through the objects in the scene. This energy is captured by sensors, which are essentially transducers converting the incident energy into electrical voltage. The voltage signal is then digitized, resulting in a digital image. This process involves sophisticated technology and precise calibration to ensure accurate representation of the physical scene in its digital form.
+
+2. Sensor Technologies and Their Role in Image Acquisition
+In digital imaging, sensor technology is pivotal. Single sensing elements, such as photodiodes, are basic forms that require movement along both x and y axes to form a two-dimensional image. In contrast, sensor strips, which are more common, capture images linearly in one direction. To obtain a complete 2D image, these strips necessitate motion in the perpendicular direction. Such technology is commonly found in devices like flatbed scanners and is also utilized in airborne imaging systems. In more specialized applications, like medical imaging (e.g., CAT scans), ring-configured sensor strips are used. These setups involve complex reconstruction algorithms to transform the captured data into meaningful images, highlighting the advanced computation involved in modern imaging technologies. Sensor arrays, like CCDs in digital cameras, consist of 2D arrays of sensing elements. They capture a complete image without requiring motion, as each element detects part of the scene. These arrays are advantageous as they don't require movement to capture an image, unlike single sensing elements and sensor strips. The captured energy is focused onto the sensor array, converted into an analog signal, and then digitized to form a digital image.

```

Comment:
it would explain what sensor technology is before tryting to explain their roles